src/quill-editor.component.ts

```diff
@@ -126,11 +127,11 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
 
     // update model if text changes
     this.quillEditor.on('text-change', (delta: any, oldDelta: any, source: string) => {
-      let html = this.editorElem.children[0].innerHTML;
+      let html: Html = this.editorElem.children[0].innerHTML;
```

Comment:
okay it was only a shot in the dark but `let html: string | null = ....` should work at all? in other cases rename the type `Html` to maybe `contentHTMLString` or something like that.

Change these things --> make a rebase and i will merge it and create a new version.

thanks